diff --git a/dist/index.js b/dist/index.js
index cf3e93b11a5af0407f1cc2cfe48b76ce91cbbe0c..090050d0c2fef7b21008d4c2149a0f1052b1ae09 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -97,6 +97,7 @@ var Thread = class {
         items.push(event.item);
       } else if (event.type === "turn.completed") {
         usage = event.usage;
+        break;
       } else if (event.type === "turn.failed") {
         turnFailure = event.error;
         break;
